tip-wlan/charts/wlan-ssc-service/templates/configmap.yaml

```diff
@@ -4,4 +4,30 @@ metadata:
   name: {{ include "common.fullname" . }}-ssc-config
   namespace: {{ include "common.namespace" . }}
 data:
+  cassandra-application.conf: |-
+    # The options in this file are overrides for the default configuration.
+    # They can also be overridden by the java system properties using -Dproperty=value
+    # For more details see https://docs.datastax.com/en/developer/java-driver/4.7/manual/core/configuration/reference/
+    datastax-java-driver {
+      basic {
+        contact-points = [ "tip-wlan-cassandra-headless:9042" ]
+        load-balancing-policy.local-datacenter = datacenter1
+        session-keyspace = tip_wlan_keyspace
+      }
+
+      advanced.ssl-engine-factory {
+        class = DefaultSslEngineFactory    
+        hostname-validation = false
+        truststore-path = /opt/tip-wlan/certs/truststore.jks
+        truststore-password = {{ .Values.creds.sslTruststorePassword }}
+        keystore-path = /opt/tip-wlan/certs/cassandra_server_keystore.jks
+        keystore-password = {{ .Values.creds.sslKeystorePassword }}
+      }
+
+      advanced.auth-provider {
+        class = PlainTextAuthProvider
+        username = tip_user
```

Comment:
Ideally even this should come from values.yaml as DUMMY_USER and then overriden in the override file.